### PR TITLE
Expand collapse all collections

### PR DESCRIPTION
**Pull request by piuccio**
Looks like this

screen shot 2015-01-19 at 14 43 49

and this

screen shot 2015-01-19 at 14 45 19

### CODE REVIEW

**Comment by michaelwmcnamara** [comment]
QA+1

**Comment by robertberry** [comment]
Neat! :+1:

File: facia-tool/public/js/widgets/fronts.html [hunk 1]
```diff
@@ -43,6 +43,12 @@
     <!-- ko ifnot: confirmSendingAlert() -->
         <div class="col__inner front-selector">
             <select data-bind="options: $root.fronts, value: front, optionsCaption: 'choose a front...'"></select>
+            <span class="collapse-expand-all" data-bind="css: {
+                expanded: allExpanded
+            }, click: toggleAll">
+                <span class="expand">expand all <i class="fa fa-chevron-up"></i></span>
+                <span class="collapse">collapse all <i class="fa fa-chevron-down"></i></span>
```
Comment (phamann) [inline facia-tool/public/js/widgets/fronts.html]: Out-of-interest, why are these `<span>`'s and not `<button>`

Comment (piuccio) [inline facia-tool/public/js/widgets/fronts.html]: Are they buttons? There's some text, so it feels like it's just a piece of text with some action...
Certainly it's easier to style a span to look like text, than a button to do the same

**Pull request merged**